### PR TITLE
Standalone IPv6 unit tests

### PR DESCRIPTION
**Pull request by guedou**
This PR adds standalone IPv6 unit tests.

### CODE REVIEW

**Comment by gpotter2** [comment]
Could you fix the conflicts?

**Comment by guedou** [comment]
Done.

Could you have a close look at these changes?

File: test/regression.uts [hunk 1]
```diff
@@ -3827,933 +1892,6 @@ p = Ether()/IPv6(dst="www.google.com")/TCP()
 assert p.dst != p[IPv6].dst
 p.show()
```
Comment (gpotter2) [inline test/regression.uts:1893]: Is there a reason you left this one?

File: test/regression.uts [hunk 2]
```diff
@@ -5924,6 +3062,7 @@ expected = {
 
 assert results_dict == expected
 
+
```
Comment (gpotter2) [inline test/regression.uts:3065]: Unnecessary. Doesn't really matter but you asked for a "close" look 😄

**Review by gpotter2 — APPROVED** [review]

File: test/regression.uts [hunk 3]
```diff
@@ -1880,1941 +1880,6 @@ assert(q[IPOption_LSRR].get_current_router() == "1.2.3.4")
...
-    s1, s2
-    tmp = inet_pton(socket.AF_INET6, "::" + s1)[8:]
-    tmp
-    assert (orb(tmp[0]) & 0x04) == 0
-    s1, s2 = in6_getRandomizedIfaceId('20b:93ff:feeb:2d3', previous=s2)
-    s1, s2
-    tmp = inet_pton(socket.AF_INET6, "::" + s1)[8:]
-    assert (orb(tmp[0]) & 0x04) == 0
-
-assert in6_getRandomizedIfaceId('20b:93ff:feeb:2d3', previous='d006:d540:db11:b092') == ('721f:11fa:3743:fc7f', '5946:5272:7fcc:108a')
```
Comment (gpotter2) [inline test/regression.uts:2126]: You should copy this test again, it has changed since the rebase.

File: test/scapy/layers/inet6.uts [hunk 4]
```diff
@@ -0,0 +1,2864 @@
...
+res=True
+for a in six.moves.range(10):
+    s1,s2 = in6_getRandomizedIfaceId('20b:93ff:feeb:2d3')
+    inet_pton(socket.AF_INET6, '::'+s1)
+    tmp2 = inet_pton(socket.AF_INET6, '::'+s2)
+    res = res and ((orb(s1[0]) & 0x04) == 0x04)
+    s1,s2 = in6_getRandomizedIfaceId('20b:93ff:feeb:2d3', previous=tmp2)
+    tmp = inet_pton(socket.AF_INET6, '::'+s1)
+    inet_pton(socket.AF_INET6, '::'+s2)
+    res = res and ((orb(s1[0]) & 0x04) == 0x04)
```
Comment (gpotter2) [inline test/scapy/layers/inet6.uts:243]: Put it here

File: test/regression.uts (hunk 3 above)
Comment (guedou) [reply test/regression.uts:2126]: Good catch! Sorry about that.

File: test/regression.uts (hunk 1 above)
Comment (guedou) [reply test/regression.uts:1893]: I was thinking to move it to l2.uts as the test name suggests.

Comment (gpotter2) [reply test/regression.uts:1893]: Makes sense 👍

**Comment by guedou** [comment]
@gpotter2 done! thanks for your review.

**Comment by guedou** [comment]
@gpotter2 are you OK with the changes?

**Review by gpotter2 — APPROVED** [review]

**Pull request merged**